src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest_getCalculableSentenceEnvelope.kt

```diff
@@ -0,0 +1,71 @@
+@file:Suppress("ClassName")
+
+package uk.gov.justice.hmpps.prison.api.resource.impl
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.http.MediaType
+import uk.gov.justice.hmpps.prison.api.model.calculation.CalculableSentenceEnvelope
+
+class BookingResourceIntTest_getCalculableSentenceEnvelope : ResourceTest() {
+
+  @Autowired
+  val objectMapper = jacksonObjectMapper()
```

Comment:
I guess this isn't actually autowired so maybe drop the annotation? Surprised it works tbh 😄